Orange/widgets/data/owdatasampler.py

```diff
@@ -226,15 +223,11 @@ def commit(self):
             if self.sampling_type in (
                     self.FixedProportion, self.FixedSize, self.Bootstrap):
                 remaining, sample = self.indices
-                self.outputInfoLabel.setText(
-                    'Outputting %d instance%s.' %
-                    (len(sample), "s" * (len(sample) != 1)))
+                self.info.set_output_summary(str(len(sample)))
             elif self.sampling_type == self.CrossValidation:
                 remaining, sample = self.indices[self.selectedFold - 1]
-                self.outputInfoLabel.setText(
-                    'Outputting fold %d, %d instance%s.' %
-                    (self.selectedFold, len(sample), "s" * (len(sample) != 1))
-                )
+                self.info.set_output_summary(str(len(sample)))
```

Comment:
This line could be moved outside the if and then the same line above can be deleted.